frontend/packages/dev-console/src/components/topology/list-view/cells/AlertsCell.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Node } from '@patternfly/react-topology';
+import { DataListCell, Tooltip } from '@patternfly/react-core';
+import { Status as TooltipStatus } from '@console/shared';
+import { pluralize } from '@console/internal/components/utils';
+import { isMobile } from '../list-view-utils';
+
+type AlertsProps = {
+  item: Node;
+};
+
+const AlertTooltip = ({ alerts, severity, noSeverityLabel = false }) => {
+  if (!alerts) {
+    return null;
+  }
+
+  const label = severity === 'Info' ? 'Message' : severity;
+  const count = alerts.length;
+  const message = _.uniq(alerts.map((a) => a.message)).join('\n');
+  const content = [
+    <span key="message" className="co-pre-wrap">
+      {message}
+    </span>,
+  ];
+
+  // Disable the tooltip on mobile since a touch also opens the sidebar, which
+  // immediately covers the tooltip content.
+  if (isMobile()) {
+    return (
+      <span className="project-overview__status">
+        <TooltipStatus
+          status={severity}
+          title={noSeverityLabel ? String(count) : pluralize(count, label)}
+        />
+      </span>
+    );
+  }
+  return (
+    <Tooltip content={content} distance={10}>
+      <span className="project-overview__status">
+        <TooltipStatus
+          status={severity}
+          title={noSeverityLabel ? String(count) : pluralize(count, label)}
+        />
+      </span>
+    </Tooltip>
+  );
```

Comment:
Nit, avoid duplication.
```suggestion
  const status = (
    <span className="project-overview__status">
      <TooltipStatus
        status={severity}
        title={noSeverityLabel ? String(count) : pluralize(count, label)}
      />
    </span>
  );

  // Disable the tooltip on mobile since a touch also opens the sidebar, which
  // immediately covers the tooltip content.
  if (isMobile()) {
    return status;
  }
  return (
    <Tooltip
      content={[
        <span key="message" className="co-pre-wrap">
          {message}
        </span>,
      ]}
      distance={10}
    >
      {status}
    </Tooltip>
  );
```